crates/swc_ecma_parser/src/parser/stmt.rs

```diff
@@ -799,6 +803,14 @@ impl<'a, I: Tokens> Parser<I> {
                 // typescript allows `declare` vars not to have initializers.
                 if self.ctx().in_declare {
                     None
+                } else if *kind == VarDeclKind::Const && self.ctx().strict {
+                    let syntax_error = if self.syntax().typescript() {
```

Comment:
❓ Do you normally consolidate errors like this that apply to ES and TS or do you keep them separate?

Reply:
I prefer consolidating